README.rst

```diff
@@ -24,6 +24,10 @@ The last stable release is available on github and can be installed with ``pip``
     $ git clone https://github.com/rqlite/pyrqlite.git
     $ pip install ./pyrqlite
 
+Often they are released on PyPI too, able to be installed via the package named ``rqlite``::
+
+    $ pip install rqlite
```

Comment:
Ah. Got it. If so, can I suggest to register pyrqlite instead?

Anyway, I am working on a version of pyrqlite compatible with the stock sqlite3 module. But lots are still missing, as types support, etc. The way it is right now there is no support for float numbers, for example.